2024/FOSDEM/osf_on_amd_5th.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Open Source Firmware status on AMD platforms 2023 - 5th edition</title>
```

Comment:
Is 2023 intentional (referring to a recap of 2023 status) or should this be 2024?

Reply:
2024 of course. Fixed. Thanks.